seldon/seldon-core-operator/base/service.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: v1
+kind: Service
+metadata:
+  labels:
+    control-plane: controller-manager
+    controller-tools.k8s.io: "1.0"
+  name: seldon-operator-controller-manager-service
+spec:
+  ports:
+  - port: 443
+  selector:
+    control-plane: controller-manager
```

Comment:
selector here is too general, it looks like no Seldon-related. why not "seldon-operator-controller-manager"?
also maybe you can add commonLabels in base/kustomization.yaml

Reply:
I changed selector. Common labels are in overlays/application as suggested by @kkasravi